### PR TITLE
Complete Question is shown now as TextView with Dropdown

### PR DESCRIPTION
**Pull request by MuhammadSalman-7214**

### CODE REVIEW

File: datacapture/src/main/res/layout/questionnaire_item_drop_down_view.xml [hunk 1]
```diff
@@ -12,19 +12,29 @@
...
+    android:orientation="vertical"
     android:layout_marginHorizontal="@dimen/item_margin_horizontal"
-    android:layout_marginVertical="@dimen/item_margin_vertical">
+    android:layout_marginVertical="@dimen/item_margin_vertical"
+    xmlns:android="http://schemas.android.com/apk/res/android">
 
-    <AutoCompleteTextView
-        android:id="@+id/auto_complete"
+    <TextView
+        android:id="@+id/dropdown_menu"
```
Comment (joiskash) [inline datacapture/src/main/res/layout/questionnaire_item_drop_down_view.xml:23]: Change this to dropdown_question_title  or something similar.

**Review by joiskash — CHANGES_REQUESTED** [review]
Good work ! Just a few comments and please fix the Unit tests too.

**Comment by jingtang10** [comment]
Thanks @MuhammadSalman-7214. Please address @joiskash 's comment and I will approve.

Additionallly, I think you'll need to fix this for the edit text layout as well: https://github.com/google/android-fhir/blob/master/datacapture/src/main/res/layout/questionnaire_item_edit_text_view.xml

Can be done in a follow-up PR.

**Comment by MuhammadSalman-7214** [comment]
sure, will complete @joiskash comments and then push the PR

**Comment by jingtang10** [comment]
> sure, will complete @joiskash comments and then push the PR

thanks! please let the reviewers know once you're done. also just a note, you won't be able to merge the PR until it is 1) approved by one the code owners and 2) all checks are passing.

File: datacapture/src/main/java/com/google/android/fhir/datacapture/views/QuestionnaireItemDropDownViewHolderFactory.kt [hunk 2]
```diff
@@ -32,7 +33,7 @@ internal object QuestionnaireItemDropDownViewHolderFactory : QuestionnaireItemVi
 ) {
     override fun getQuestionnaireItemViewHolderDelegate() =
         object : QuestionnaireItemViewHolderDelegate {
-            private lateinit var textInputLayout: TextInputLayout
+            private lateinit var textInputLayout: TextView
```
[the remark this answers is not in the document]
Comment (MuhammadSalman-7214) [reply datacapture/src/main/java/com/google/android/fhir/datacapture/views/QuestionnaireItemDropDownViewHolderFactory.kt:36]: Done and Unit tests done too

File: datacapture/src/main/res/layout/questionnaire_item_drop_down_view.xml (hunk 1 above)
Comment (MuhammadSalman-7214) [reply datacapture/src/main/res/layout/questionnaire_item_drop_down_view.xml:23]: Done

**Comment by MuhammadSalman-7214** [comment]
Changes done @jingtang10 and @joiskash. Both of you can review the changes.

File: datacapture/src/androidTest/java/com/google/android/fhir/datacapture/views/QuestionnaireItemDropDownViewHolderFactoryInstrumentedTest.kt [hunk 3]
```diff
@@ -66,8 +66,8 @@ class QuestionnaireItemDropDownViewHolderFactoryInstrumentedTest {
         )
 
         assertThat(
-            viewHolder.itemView.findViewById<TextInputLayout>(R.id.dropdown_menu)
-                .hint
+            viewHolder.itemView.findViewById<TextView>(R.id.dropdown_question_title)
+                .text
```
Comment (jingtang10) [inline datacapture/src/androidTest/java/com/google/android/fhir/datacapture/views/QuestionnaireItemDropDownViewHolderFactoryInstrumentedTest.kt:70]: same line

File: datacapture/src/androidTest/java/com/google/android/fhir/datacapture/views/QuestionnaireItemDropDownViewHolderFactoryInstrumentedTest.kt [hunk 4]
```diff
@@ -46,7 +46,7 @@ class QuestionnaireItemDropDownViewHolderFactoryInstrumentedTest {
     @Before
     fun setUp() {
         context = ContextThemeWrapper(
-            InstrumentationRegistry.getInstrumentation().getTargetContext(),
+            InstrumentationRegistry.getInstrumentation().targetContext,
```
Comment (jingtang10) [inline datacapture/src/androidTest/java/com/google/android/fhir/datacapture/views/QuestionnaireItemDropDownViewHolderFactoryInstrumentedTest.kt:49]: thanks for fixing this -- can you do this for other files too?

**Review by jingtang10 — CHANGES_REQUESTED** [review]

**Comment by jingtang10** [comment]
Thanks - just a couple of nits

File: datacapture/src/androidTest/java/com/google/android/fhir/datacapture/views/QuestionnaireItemDropDownViewHolderFactoryInstrumentedTest.kt (hunk 3 above)
Comment (MuhammadSalman-7214) [reply datacapture/src/androidTest/java/com/google/android/fhir/datacapture/views/QuestionnaireItemDropDownViewHolderFactoryInstrumentedTest.kt:70]: okay

File: datacapture/src/androidTest/java/com/google/android/fhir/datacapture/views/QuestionnaireItemDropDownViewHolderFactoryInstrumentedTest.kt (hunk 4 above)
Comment (MuhammadSalman-7214) [reply datacapture/src/androidTest/java/com/google/android/fhir/datacapture/views/QuestionnaireItemDropDownViewHolderFactoryInstrumentedTest.kt:49]: okay

File: datacapture/src/androidTest/java/com/google/android/fhir/datacapture/views/QuestionnaireItemDropDownViewHolderFactoryInstrumentedTest.kt (hunk 3 above)
Comment (MuhammadSalman-7214) [reply datacapture/src/androidTest/java/com/google/android/fhir/datacapture/views/QuestionnaireItemDropDownViewHolderFactoryInstrumentedTest.kt:70]: Done

File: datacapture/src/androidTest/java/com/google/android/fhir/datacapture/views/QuestionnaireItemDropDownViewHolderFactoryInstrumentedTest.kt (hunk 4 above)
Comment (MuhammadSalman-7214) [reply datacapture/src/androidTest/java/com/google/android/fhir/datacapture/views/QuestionnaireItemDropDownViewHolderFactoryInstrumentedTest.kt:49]: Done

**Comment by MuhammadSalman-7214** [comment]
@jingtang10 both nits done

**Review by jingtang10 — APPROVED** [review]
Thanks @MuhammadSalman-7214!

🚀

**Comment by MuhammadSalman-7214** [comment]
your welcome @jingtang10 :-)

**Pull request merged**